From 97b5dcaa6b221eb403cc92e953225d38aee18f70 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 1 Sep 2022 14:48:39 -0700
Subject: [PATCH] Match reutrn type from
 RAPTOR_ASSERT_OBJECT_POINTER_RETURN_VALUE

This ensures that integer type 0 is returned and not NULL
Fixes
raptor_serialize.c:243:66: error: incompatible pointer to integer conversion returning 'void *' from a function with result type 'int' [-Wint-conversion]
  RAPTOR_ASSERT_OBJECT_POINTER_RETURN_VALUE(world, raptor_world, NULL);
                                                                 ^~~~
Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/raptor_parse.c     | 2 +-
 src/raptor_serialize.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/raptor_parse.c b/src/raptor_parse.c
index 26911f4..0091e1e 100644
--- a/src/raptor_parse.c
+++ b/src/raptor_parse.c
@@ -257,7 +257,7 @@ raptor_world_get_parser_factory(raptor_world *world, const char *name)
 int
 raptor_world_get_parsers_count(raptor_world* world)
 {
-  RAPTOR_ASSERT_OBJECT_POINTER_RETURN_VALUE(world, raptor_world, NULL);
+  RAPTOR_ASSERT_OBJECT_POINTER_RETURN_VALUE(world, raptor_world, 0);
 
   raptor_world_open(world);
 
diff --git a/src/raptor_serialize.c b/src/raptor_serialize.c
index a1f29d7..2bf4ab2 100644
--- a/src/raptor_serialize.c
+++ b/src/raptor_serialize.c
@@ -240,7 +240,7 @@ raptor_get_serializer_factory(raptor_world* world, const char *name)
 int
 raptor_world_get_serializers_count(raptor_world* world)
 {
-  RAPTOR_ASSERT_OBJECT_POINTER_RETURN_VALUE(world, raptor_world, NULL);
+  RAPTOR_ASSERT_OBJECT_POINTER_RETURN_VALUE(world, raptor_world, 0);
 
   raptor_world_open(world);
 
-- 
2.37.3

